webpack.config.js

```diff
@@ -8,9 +8,24 @@ const CopyWebpackPlugin = require("copy-webpack-plugin");
 const { merge } = require("webpack-merge");
 const loadPreset = require("./config/presets/loadPreset");
 const loadConfig = (mode) => require(`./config/webpack.${mode}.js`)(mode);
+const fs = require("fs");
 
 module.exports = function (env) {
   const { mode = "production" } = env || {};
+
+  let hasDefaultRoute = false;
+  let defaultRoute = {};
+  // Read bos.config.json
+  try {
+    const bosConfig = JSON.parse(
+      fs.readFileSync("../bos.config.json", "utf-8")
```

Comment:
Maybe include your `bos.config.json` as an example of what it should look like?

Also would be great to include some info about this in README.md